python/mxnet/module/module.py

```diff
@@ -7,9 +7,9 @@
 import logging
 import warnings
 
+import mxnet as mx
```

Comment:
You are right. What I meant was to update your branch to the latest dmlc/sparse so that this line would be gone. Otherwise, you may not running tests using your latest changes.